### ISSUE TITLE
Login Issue

### ISSUE THREAD

**Issue by BryantJDaniel** [64Soft/NRatings.Client#47]
![Issue](https://user-images.githubusercontent.com/132112932/235273561-1c9461e0-8255-4d5c-a483-3e6f3c2e8589.png)

Not sure why I cant login Correctly, Have tried redownloading, and making sure I have a correct default browser.

**Comment by 64Soft** [comment]
Strange, just tested now, works correctly here. Maybe it was a temporary issue with the auth0.com services. Can you retry ?

**Comment by BryantJDaniel** [comment]
Still giving me the same error

**Comment by ai-line-mod** [comment]
@64Soft From reddit:

"Alright so I downloaded Chrome and made it my default browser. I opened up NRatings, hit tools->options, and then hit log in. This time it let me log in and enter my email and password and stuff. Which I did and it said log in completed. But when I went back into NRatings the same error message appeared."

A default browser is set, it allows a login, but for some reason the credentials aren't being passed back to the client? I have not been able to replicate this myself, nor am I certain where the exact issue could lie.
